### PR TITLE
[IMP] In the bank statement reconcile wizard, display the line note in the details popup

### PR DESCRIPTION
**Pull request by lmignon**
When reconciling, it's sometimes useful to have access to the note since the note can be filled with some information by the parser.

### CODE REVIEW

**Comment by Cedric-Pigeon** [comment]
:+1:

**Comment by sbidoul** [comment]
rebased on an older commit

File: addons/account/static/src/xml/account_bank_statement_reconciliation.xml [hunk 1]
```diff
@@ -172,6 +173,7 @@
                 <t t-if="line.debit !== 0" t-esc="line.debit_str"/><t t-if="line.credit !== 0" t-esc="line.credit_str"/>
                 <t t-if="line.amount_currency_str"> (<t t-esc="line.amount_currency_str"/>)</t>
             </td></tr>
+             <tr t-if="line.note"><td>Note</td><td><t t-esc="line.note"/></td></tr>
```
Comment (Whisno) [inline addons/account/static/src/xml/account_bank_statement_reconciliation.xml]: Ok about the first note addition, but isn't this one a mistake ? I think this object has no note attribute : https://github.com/odoo/odoo/blob/8.0/addons/account/account_move_line.py#L760

**Comment by lmignon** [comment]
@Whisno you are right.

**Comment by Whisno** [comment]
Ok, then it's settled 2faf4b7747310a58b93eea5d3d2d7ab0bf06ca33 (forgot to add 'Closes ...')
Thanks for the contribution

**Comment by lmignon** [comment]
@Whisno thank you

**Pull request closed without merging**